jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/timers/DslConstantThroughputTimer.java

```diff
@@ -0,0 +1,78 @@
+package us.abstracta.jmeter.javadsl.core.timers;
+
+import java.lang.reflect.Method;
+import java.util.List;
+import org.apache.jmeter.testbeans.gui.TestBeanGUI;
+import org.apache.jmeter.testelement.TestElement;
+import org.apache.jmeter.timers.ConstantThroughputTimer;
+import us.abstracta.jmeter.javadsl.codegeneration.MethodCall;
+import us.abstracta.jmeter.javadsl.codegeneration.MethodCallContext;
+import us.abstracta.jmeter.javadsl.codegeneration.SingleTestElementCallBuilder;
+import us.abstracta.jmeter.javadsl.codegeneration.TestElementParamBuilder;
+
+/**
+ * Allows using JMeter Constant Throughput Timers which pauses samplers under it's control to
+ * achieve a target number of samples per minute.
+ * <p>
+ * The number of threads must be sufficient to enable the target samples per minute to be
```

Comment:
great doc! :)

Reply:
Can you also add a section in user guide (docs/guilde/README.md) explaining when to use this element (and when to use rpsThread group for example) and provide an example?

Reply:
Not done this yet as had a busy week!

Reply:
@rabelenda updated the readme now